### PR TITLE
[OPAL-429][MNOE-1113] Add scoping of subscription_event to organization

### PR DESCRIPTION
**Pull request by iseessel**

### CODE REVIEW

File: api/config/routes.rb [hunk 1]
```diff
@@ -281,11 +281,11 @@
           resources :teams, only: [:index]
 
           if Settings&.dashboard&.marketplace&.provisioning
-            resources :subscriptions, only: [:index, :show, :create, :update] do
-              member do
-                post :cancel
-              end
+            resources :subscription_events, only: [:index]
```
Comment (BrunoChauvet) [inline api/config/routes.rb]: 👍

File: api/config/routes.rb [hunk 2]
```diff
@@ -281,11 +281,11 @@
...
+          end
 
+          if Settings&.dashboard&.marketplace&.provisioning
```
Comment (BrunoChauvet) [inline api/config/routes.rb]: You can group this within the same `if Settings&.dashboard&.marketplace&.provisioning` above

Comment (iseessel) [reply api/config/routes.rb]: lol oops thanks.

File: api/lib/mno_enterprise/concerns/controllers/jpi/v1/admin/subscription_events_controller.rb [hunk 3]
```diff
@@ -18,11 +18,13 @@ def index
     # Fetch all the subscription events of an organization
     elsif params[:organization_id]
       org = MnoEnterprise::Organization.includes([:subscriptions]).find(params[:organization_id]).first
+
       #Find organization's subscription_ids.
       query = MnoEnterprise::SubscriptionEvent
         .apply_query_params(params)
-        .where('subscription.id': org.subscriptions.map(&:id))
+        .where('subscription.id': org.subscriptions.map(&:id).presence)
```
Comment (iseessel) [inline api/lib/mno_enterprise/concerns/controllers/jpi/v1/admin/subscription_events_controller.rb]: https://github.com/JsonApiClient/json_api_client/issues/283

**Pull request merged**